torchgen/dest/lazy_ir.py

```diff
@@ -222,6 +241,11 @@ class {schema.node_name} : public {self.node_base} {{
     {has_optional_defs}
   }}
 
+  bool Equal({node_ctor_args}) const {{
```

Comment:
Can we make this part of codegen optional? PyTorch/xla can't use it yet until we migrate to LazyTensor Class